resources/views/bread/browse.blade.php

```diff
@@ -13,6 +13,7 @@
             </a>
         @endcan
         @can('delete', app($dataType->model_name))
+            @php $showCheckboxColumn = true @endphp
```

Comment:
seems weird setting a value of `true` then using `@isset()` below.  Perhaps give it a default value of false then juse `@if()` it instead....? Doesn't really save any code, but I think it reads cleaner.  Thoughts?

Reply:
I agree it's going to be more readable and it only adds one line of code.
I'll submit the changes asap.